### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ArtMot99**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(list_of_dict: list) -> list:
+    # """
+    # Create Person List Function
+    #
+    # This function create list of instances of Person class
+    # """
```
Comment (Abnormaltype) [inline app/main.py:15]: use comments or docstring)

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
+    list_of_people = []
+    for person in list_of_dict:
+        new_person = Person(person["name"], person["age"])
+        if person.get("wife") is not None:
+            new_person.wife = person["wife"]
+        if person.get("husband") is not None:
+            new_person.husband = person["husband"]
+        list_of_people.append(new_person)
+    for temp in list_of_people:
```
Comment (Abnormaltype) [inline app/main.py:24]: temp is uninformative name for a variable

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
also, you should write your code. copy paste it's a bad way

**Comment by ArtMot99** [comment]
This is not a copy, we worked on the code together, used all the features of GitHub 👌🏻

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
but why does the difference in your PR have a lot of time?)

**Comment by ArtMot99** [comment]
It so happened) In the future we will upload the code together, I hope this will not be a problem?

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
my advice: try to write different code)

**Comment by ArtMot99** [comment]
Understood accepted 👌🏻

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
fix my comments please)

**Pull request closed without merging**